velox/functions/prestosql/aggregates/SumAggregate.cpp

```diff
@@ -112,7 +112,8 @@ exec::AggregateRegistrationResult registerSum(
         }
       },
       withCompanionFunctions,
-      overwrite);
+      overwrite,
+      {false /*orderSensitive*/});
```

Comment:
FYI, sum(double) is order sensitive because the amount of error accumulated depends on the order of inputs. Not saying you should change this flag, but just want to mention this caveat.